web/viewer.html

```diff
@@ -263,30 +263,39 @@
                 <span id="numPages" class="toolbarLabel"></span>
               </div>
               <div id="toolbarViewerRight">
-                <button id="presentationMode" class="toolbarButton hiddenLargeView" title="Switch to Presentation Mode" tabindex="31" data-l10n-id="presentation_mode">
+                <div id="editionModeButtons" class="splitToolbarButton hidden" role="radiogroup">
+                  <button id="editorSelect" class="toolbarButton" title="Enter in edit mode" role="radio" aria-checked="false" tabindex="31" data-l10n-id="editor_select">
+                    <span data-l10n-id="editor_select_label">Edit Mode</span>
+                  </button>
+                  <button id="editorFreeText" class="toolbarButton" title="Add some free text" role="radio" aria-checked="false" tabindex="32" data-l10n-id="editor_free_text">
+                    <span data-l10n-id="editor_free_text_label">Free Text Annotation</span>
+                  </button>
+                </div>
+
+                <button id="presentationMode" class="toolbarButton hiddenLargeView" title="Switch to Presentation Mode" tabindex="33" data-l10n-id="presentation_mode">
```

Comment:
How many new buttons do you foresee adding?

To avoid changing the blame again, can you please increase this and the following `tabindex`s to account for additional buttons above such that we don't need to touch them again in just a couple of days?